.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
```

Comment:
Great job @gfrancoarq 🤝

- [ ] Great job adding the `linters.yml` file. However, for this project, you need to add both the `.hintrc` file and the `.stylelintrc.json` file. Remember to add these two files to the root directory. Please find the correct [linter files in this link](https://github.com/microverseinc/linters-config/tree/master/html-css).

Reply:
I added the files. I'll wait for your comments. Thank you!